src/org/omegat/util/Preferences.java

```diff
@@ -670,9 +670,9 @@ public static void setSRX(SRX newSrx) {
         SRX oldValue = srx;
         srx = newSrx;
 
-        File srxFile = new File(StaticUtils.getConfigDir() + SRX.CONF_SENTSEG);
+        File srxDir = new File(StaticUtils.getConfigDir());
         try {
-            SRX.saveTo(srx, srxFile);
+            SRX.saveTo(srx, srxDir);	// defaults to SRX format, but if conf file exists, overwrite it
```

Comment:
I don't think this comment is accurate w.r.t. overwriting the conf.

Reply:
True, this comment was coherent with the first version of my patches, where I tried to find a way to maintain both formats. But during our discussions it became clear that you want the old format to disappear, so I removed any possibility to save to old conf format. 
Now it saves always and only to SRX.